LSPS1/extensions/refunds/LSPS1.refunds.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.3
+info:
+  description: |
+    API documentation for channel opening request from a Lightning Network Service Provider (LSP). Refunds.
+    
+  version: "1"
+  title: "LSP Spec 1"
+tags:
+- name: "Refund"
+  description: "Extension: `refunds`"
```

Comment:
W/hy is this an extension rather than part of base?

Reply:
Refunds are still a todo. Ignore all extensions for the moment.

Reply:
Cannot really do onchain without refunds.  Yes, you can add all sorts of stuff to make sure you "never" run out of money to service incoming requests, but accidents happen and refund mechanisms are necessary.  Either cut out onchain completely from base and put onchain+refund on an extension, or have them both in base.  I suggest have them both in base.

Reply:
Again, this is on my todo list. Will have a look at it next week.

Reply:
Please have a look at my update in the PR and let me know if you agree with it.